components/Cards/src/CardThemer/MDCOutlinedCardThemer.h

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCardScheme.h"
+
+#import "MaterialCards.h"
+
+#import <Foundation/Foundation.h>
+
+/** Applies material style to outlined MDCCard objects. */
+@interface MDCOutlinedCardThemer : NSObject
```

Comment:
Outlined cards are a variant rather than a first-class component (e.g. Contained Buttons, Text Buttons). The outline variants should therefore be implemented similarly to the "surface" variants for app bar and friends.